bed/methods.go

```diff
@@ -48,7 +48,7 @@ func (b BedSlice) Write(file string) {
 
 func (b *Bed) WriteToFileHandle(file *fileio.EasyWriter) {
 	//TODO: write max fields that are non-nil?
-	WriteToFileHandle(file, b, 7)
```

Comment:
Why was it "7" before?

Reply:
The Bed format has a number of optional fields after the main positional information which may or may not be initialized in the data structure. The old version of WriteToFileHandle specifies the number of fields to write as a hardcoded number but here we actually check how many fields are initialized in the struct (I added a Fields component to the Bed struct to track this) so we can have a more flexible write function.